### PR TITLE
Bugfixes for PHP 5.6

### PR DESCRIPTION
**Pull request by pawelchcki**
Fix 5.6 instrumentation dispatch bugs. 
+ reenable tests disabled for php 5.6
+ add dockerfiles that build debug version of PHP - extremely useful for tracing

### CODE REVIEW

File: tests/Integration/Integrations/MemcachedTest.php [hunk 1]
```diff
@@ -26,10 +26,6 @@ protected function setUp()
     {
         parent::setUp();
 
-        if (PHP_MAJOR_VERSION < 7) {
```
Comment (labbati) [inline tests/Integration/Integrations/MemcachedTest.php]: `Predis` integration tests (just recently merged to master), they were also skipped, so you may want to pull master and re-enabled them as well

File: docker-compose.yml [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
-    tmpfs: [ '/home/circleci/app/tmp:uid=3434,gid=3434,exec' ]
-    depends_on:
-      - memcached_integration
-      - redis_integration
-      - ddagent_integration
-    environment:
-      - REDIS_HOSTNAME=redis_integration
-      - DDAGENT_HOSTNAME=ddagent_integration
+x-aliases:
+  - &base_php_service
```
Comment (labbati) [inline docker-compose.yml]: 💯

File: docker-compose.yml [hunk 3]
```diff
@@ -1,29 +1,32 @@
...
+      ulimits:
+        core: 99999999999
+      working_dir: '/home/circleci/app'
+      stdin_open: true
+      tty: true
+      volumes: [ '.:/home/circleci/app' ]
+      tmpfs: [ '/home/circleci/app/tmp:uid=3434,gid=3434,exec' ]
+      depends_on:
+        - redis_integration
```
Comment (labbati) [inline docker-compose.yml]: Just adding this comment to make sure that leaving memcached out was intentional. (It is not used though)

File: tests/Integration/Integrations/MemcachedTest.php (hunk 1 above)
Comment (pawelchcki) [reply tests/Integration/Integrations/MemcachedTest.php]: Ok will reenable them 🤞

File: docker-compose.yml (hunk 3 above)
Comment (pawelchcki) [reply docker-compose.yml]: Probably wrong merge, I'll re-add it

File: tests/Integration/Integrations/MemcachedTest.php (hunk 1 above)
Comment (pawelchcki) [reply tests/Integration/Integrations/MemcachedTest.php]: Phew, tests have passed as well 🕺

**Pull request merged**